move unique_key page under general configs (#6636)
fishtown-analytics/docs.getdbt.com: website/docs/reference/resource-configs/unique_key.md

---
resource_types: [snapshots, models]
description: "Learn more about unique_key configurations in dbt."
datatype: column_name_or_expression
---

<Tabs>

<TabItem value="models" label="Models">

> Configure the `unique_key` in the `config` block of your [incremental model](/docs/build/incremental-models) SQL file, in your `models/properties.yml` file, or in your `dbt_project.yml` file.

Reviewer: ```suggestion
Configure the `unique_key` in the `config` block of your [incremental model's](/docs/build/incremental-models) SQL file, in your `models/properties.yml` file, or in your `dbt_project.yml` file.
```

## Examples
### Use an `id` column as a unique key

<Tabs>

<TabItem value="models" label="Models">

> In this example, we use an `id` column as a unique key for an incremental model.

Reviewer: ```suggestion
This example uses an `id` column as a unique key for an incremental model.
```

> In this example, we use an `id` column as a unique key for an incremental model.

Reviewer: To avoid switching from "you" to "we" in the text
```suggestion
In this example, the `id` column is the unique key for an incremental model.
```

<File name='models/my_incremental_model.sql'>

```sql
{{
    config(
        materialized='incremental',
        unique_key='id'
    )
}}

select * from ..
```

</File>
</TabItem>

<TabItem value="snapshots" label="Snapshots">

> In this example, we use an `id` column as a unique key for a snapshot.

Reviewer: ```suggestion
In this example, the `id` column is used as a unique key for a snapshot.
```

</File>

</TabItem>
</Tabs>

<VersionBlock firstVersion="1.9">

### Use multiple unique keys

<Tabs>
<TabItem value="models" label="Models">

> Configure multiple unique keys for an incremental model as a string representing a single column or a list of single-quoted column names that can be used together, for example, `['col1', 'col2', …]`. 

Reviewer: ```suggestion
Configure multiple unique keys for an incremental model as a string representing a single column or a list of single-quoted column names that can be used together. For example, `['col1', 'col2', …]`. 
```
